playbooks/cluster_setup_network.yaml

```diff
@@ -141,10 +141,9 @@
   vars:
     apply_config: "{{ apply_network_config | default(false) }}"
   tasks:
-    - name: Restart systemd-networkd
-      ansible.builtin.systemd:
-        name: systemd-networkd
-        state: restarted
+    - name: Reload systemd-networkd configuration
+      command:
```

Comment:
Why using command instead of systemd plugin?
The systemd supported the state "reloaded".
https://docs.ansible.com/ansible/2.9/modules/systemd_module.html

Reply:
ok